views/partials/prs.hbs

```diff
@@ -14,6 +14,23 @@
 
   <div class="row">
     <div class="large-12 columns">
+      <div class="social-head">
+        <h4><strong>Share:</strong></h4>
+        <div class="social">
+          <ul>
+            <li><div class="twitter"><a href="https://twitter.com/intent/tweet?button_hashtag=hacktoberfest&text={{ prs.length }} / 4 {{ prs.statement }}" target="_blank">Twitter</a></div></li>
```

Comment:
Can we tweak the message to something like "I'm on 3/4 pull requests, how about you? #hacktoberfest"?